lib/pages/editor-page.js

```diff
@@ -140,4 +140,18 @@ export default class EditorPage extends BaseContainer {
 	titleShown() {
 		return this.driver.findElement( by.css( '.editor-title__input' ) ).getAttribute( 'value' );
 	}
+
+	publishEnabled() {
+		const publishSelector = by.css( '.editor-publish-button' );
+		return this.driver.findElement( publishSelector ).getAttribute( 'disabled' ).then( ( d ) => {
+			return d !== 'true';
+		} );
+	}
+
+	emailVerificationNotice() {
+		const emailVerificationSelector = by.css( '.editor-ground-control__email-verification-notice' );
+		return driverHelper.isElementPresent( this.driver, emailVerificationSelector ).then( ( present ) => {
+			return present;
```

Comment:
This seems a little redundant.  Can't we just `return driverHelper.isElementPresent( this.driver, emailVerificationSelector )` and leave out the `.then()`?